packages/custom-functions-metadata/test/cases/error-linkedentitydataprovider-excludefromautocomplete/functions.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+/**
+ * Test the linkedEntityDataProvider tag in combination with the excludeFromAutoComplete tag
+ * @param linkedEntityId {any} Unique `LinkedEntityId` of the `LinkedEntityCellValue`s which is being requested for resolution/refresh.
```

Comment:
I think for JS the @param can just omit the `{any}` part since it doesn't need to convey type info for the parameter. Let me know if that doesn't work.